testdata/filters/stackdriver_outbound.yaml.tmpl

```diff
@@ -18,7 +18,7 @@
           "@type": "type.googleapis.com/google.protobuf.StringValue"
           value: |
             {{- if .Vars.JustSendErrorClientLog }}
```

Comment:
Can we replace this condition with `if .Vars.StackdriverFilterClientErrorConfig`

Reply:
Fixed